### PR TITLE
[6CORE-005] added skeleton of Delivery Management service

### PR DESCRIPTION
**Pull request by YuraKhilko**
Created skeleton of Delivery Management service: parent module delivery and child modules: app, boot, domain, infra.

### CODE REVIEW

File: 3-platform/delivery/.gitignore [hunk 1]
```diff
@@ -0,0 +1,38 @@
+target/
```
Comment (sokimaaa) [inline 3-platform/delivery/.gitignore]: there is already a present gitignore in the root folder (./6core/.gitignore). 
Modify it for reusing purposes.

File: 3-platform/delivery/app/pom.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </parent>
+
+    <artifactId>app</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.6core.platform</groupId>
+            <artifactId>domain</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+        </dependency>
```
Comment (sokimaaa) [inline 3-platform/delivery/app/pom.xml]: GroupId and ArtifactId will overlap in the future with other services and maven won't understand which artifact to get in project.

Consider renaming your groupId/artifactId to be unique.
The same for app, infra, boot.
for example:
```
        <dependency>
            <groupId>com.6core.platform</groupId>
            <artifactId>delivery-domain</artifactId>
            <version>0.0.1-SNAPSHOT</version>
        </dependency>
```

**Review by sokimaaa — CHANGES_REQUESTED** [review]

File: 3-platform/delivery/.gitignore (hunk 1 above)
Comment (YuraKhilko) [reply 3-platform/delivery/.gitignore]: @sokimaaa , does it mean that we'll have only one .gitignore file for all microservices?

**Review by kateryna-mykh — APPROVED** [review]
Approved.

**Review by sokimaaa — APPROVED** [review]

**Pull request merged**